client/my-sites/customer-home/controller.jsx

```diff
@@ -10,12 +10,19 @@ import page from 'page';
  */
 import CustomerHome from './main';
 import { getSelectedSiteSlug, getSelectedSiteId } from 'state/ui/selectors';
-import { canCurrentUserUseCustomerHome } from 'state/sites/selectors';
+import { canCurrentUserUseCustomerHome, getSitePlan } from 'state/sites/selectors';
 import isRecentlyMigratedSite from 'state/selectors/is-site-recently-migrated';
+import reducer from 'state/concierge/reducer';
+
+export default async function( context, next ) {
+	const state = await context.store.getState();
+	const siteId = await getSelectedSiteId( state );
+	const plan = await getSitePlan( state, siteId );
+
+	if ( ! plan.is_free && [ 'Business', 'eCommerce' ].includes( plan.product_name_short ) ) {
```

Comment:
Can we maybe use `isBusinessPlan` and `isEcommercePlan`?

https://github.com/Automattic/wp-calypso/blob/c78781876f716e094f9628c5c514a469f32508f6/client/lib/plans/index.js#L217-L223

Reply:
Oooh, yes, good idea 👍 9f1ab155c1